Mapping/ContainerEntityListenerResolver.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+namespace Doctrine\Bundle\DoctrineBundle\Mapping;
+
+use InvalidArgumentException;
+use Psr\Container\ContainerInterface;
+use RuntimeException;
+
+class ContainerEntityListenerResolver implements EntityListenerServiceResolver
+{
+    /** @var ContainerInterface */
```

Comment:
This php doc could be removed

Reply:
No: while the property is assigned a value in the constructor, thus implicitly setting a type, static analysis can't assume that this is the only allowed type. To have this information for static analysis, we still keep `@var` around until we've got typed properties.

Reply:
Thank you 👍🏻

Reply:
Actually need to revert this :smile: 

```
 11 | ERROR | [ ] Property
    |       |     \Doctrine\Bundle\DoctrineBundle\Mapping\ContainerEntityListenerResolver::$container
    |       |     does not have @var annotation.
    |       |     (SlevomatCodingStandard.TypeHints.TypeHintDeclaration.MissingPropertyTypeHint)
```